src/main/java/com/github/manolo8/darkbot/gui/tree/editors/ImagePicker.java

```diff
@@ -2,40 +2,26 @@
 
 import com.github.manolo8.darkbot.config.ImageWrapper;
 import com.github.manolo8.darkbot.gui.AdvancedConfig;
+import com.github.manolo8.darkbot.gui.utils.Strings;
+import com.github.manolo8.darkbot.gui.utils.window.FileChooserUtil;
 import eu.darkbot.api.config.ConfigSetting;
 import eu.darkbot.api.config.util.OptionEditor;
-import java.awt.Component;
 import java.awt.Dimension;
-import java.awt.HeadlessException;
-import java.io.File;
 import javax.swing.JButton;
 import javax.swing.JComponent;
-import javax.swing.JDialog;
 import javax.swing.JFileChooser;
 import javax.swing.filechooser.FileNameExtensionFilter;
 
+
 public class ImagePicker extends JButton implements OptionEditor<ImageWrapper> {
     private JFileChooser fc;
     private ImageWrapper image;
 
     public ImagePicker() {
         addActionListener(e -> {
-            if (fc == null) {
-                String path = image == null ? new File(System.getProperty("user.dir")).getAbsolutePath() : image.getPath();
-                fc = new JFileChooser(path) {
-                    @Override
-                    protected JDialog createDialog(Component parent) throws HeadlessException {
-                        JDialog dialog = super.createDialog(parent);
-                        dialog.setAlwaysOnTop(true);
-                        return dialog;
-                    }
-                };
-                FileNameExtensionFilter filter = new FileNameExtensionFilter(
-                        "Images Files", "png", "jpeg", "jpg", "gif");
-                fc.setFileFilter(filter);
-                fc.setAcceptAllFileFilterUsed(false);
-            }
-
+            if (fc == null) fc = FileChooserUtil.getChooser(image.getPath(),
+                    new FileNameExtensionFilter("Images Files", "png", "jpeg", "jpg", "gif"),
+                    false);
```

Comment:
```suggestion
            if (fc == null)
                fc = FileChooserUtil.getChooser(image.getPath(),
                        new FileNameExtensionFilter("Images Files", "png", "jpeg", "jpg", "gif"),
                        false);
```
unless it fits in a single line, i think it's best to always put the continuation of the if in a new line for readability